packages/dashboard/src/components/robots/robot-page.tsx

```diff
@@ -37,24 +87,44 @@ export function RobotPage() {
     const slicedRobots = robots.slice(0, 10);
     setVerboseRobots(slicedRobots);
     return slicedRobots;
-  }, [fleetsApi, page]);
+  }, [rmfIngress, page]);
 
   React.useEffect(() => {
     fetchVerboseRobots();
   }, [fetchVerboseRobots]);
 
   return (
-    <RobotPanel
-      className={classes.robotPanel}
-      fetchVerboseRobots={fetchVerboseRobots}
-      paginationOptions={{
-        count: hasMore ? -1 : page * 10 + verboseRobots.length,
-        rowsPerPage: 10,
-        rowsPerPageOptions: [10],
-        page,
-        onChangePage: (_ev, newPage) => setPage(newPage),
-      }}
-      verboseRobots={verboseRobots}
-    />
+    <Grid container className={classes.container}>
+      <Grid item xs={3}>
```

Comment:
What are your thoughts on making the space for the map bigger? 4 or 5, cos I'm thinking if the map is bigger like the clinic or hotel world, a larger space for the map would provide better visualization

![image](https://user-images.githubusercontent.com/28668944/140447710-49be407f-97d2-492d-b3bb-3a23b61751ae.png)
this is with `xs={5}`

Reply:
Changed the xs value to 4 as the table at 5 looks too small on my laptop. @koonpeng  should we remove the table `min-width` value in favor of a bigger map?

Reply:
The screenshoot above looks weird, I feel there is no panels that clearly shows the center of attention.

We can reduce the min width, but the table should be the center of attention, so it shouldn't be too small that the map takes away the attention.

Does the head post contains the latest changes? If not, can you update them regularly?

Reply:
I think the screen shot above is just to show the bigger map size and not the zooming in. Updated the head post with the latest screenshot